src/swap/SwapScreen.tsx

```diff
@@ -42,7 +44,19 @@ const DEFAULT_SWAP_AMOUNT: SwapAmount = {
 
 const { decimalSeparator } = getNumberFormatSettings()
 
-export function SwapScreen() {
+type Props = NativeStackScreenProps<StackParamList, Screens.SwapScreenWithBack>
+
+function SwapScreen() {
+  return <SwapScreenSection showDrawerTopNav={true} />
+}
+
+export function SwapScreenSection({
+  showDrawerTopNav,
+  props,
+}: {
+  showDrawerTopNav: boolean
+  props?: Props
```

Comment:
could this explicitly be a "navigation" object? We don't need all the props of the SwapScreenWithBack component right?

Reply:
Good call, changed it to that